CRM/Admin/Form/Setting.php

```diff
@@ -150,7 +155,9 @@ public function buildQuickForm() {
 
       }
     }
+    // setting_description should be deprecated - see Mail.tpl for metadata based tpl.
     $this->assign('setting_descriptions', $descriptions);
+    $this->assign('settings', $settingMetaData);
```

Comment:
isn't **settingFields** would be more descriptive name instead of just settings ?